README.md

```diff
@@ -61,26 +67,36 @@ quay.io/redhat-developer/gitops-backend:v0.0.1
 If that's all what you are changing, the following steps are not needed in development
 mode. You could update your image "payload" and re-install the operator.
 
-* Build the operator image.
+Set the base image and version for building operator, bundle and index images.
+
+```
+export IMAGE=quay.io/redhat-developer/gitops-backend-operator VERSION=0.0.4
+```
```

Comment:
@iam-veeramalla I've updated the `Makefile` to work with a single image variable. Exporting this is sufficient for both operator and bundle images

Reply:
Thanks 🙏